### PR TITLE
Landing icons

### PR DESCRIPTION
**Pull request by NateBaldwinDesign**
## Summary
Added icon images to home page sections to give a little extra visual interest

## Additional notes
image

### CODE REVIEW

**Comment by NateBaldwinDesign** [comment]
@lazd any ideas what's causing this? 
image

When I test locally, all tests pass (using `yarn test`)
image

**Comment by hyperknot** [comment]
> @lazd any ideas what's causing this?
> image
> 
> When I test locally, all tests pass (using `yarn test`)
> image

GitHub tests have been unreliable lately, it happens on many PR.

**Comment by jzohrab** [comment]
Hi @NateBaldwinDesign - maybe this one is good to go into FE.  Perhaps ping @lazd.  Cheers! jz

File: covidatlas/index.css [hunk 1]
```diff
@@ -144,10 +144,13 @@ sp-dropdown[emphasized] {
 
 .ca-Landing {
   background-color: var(--BackgroundColor);
+  background-image: url('logo-outlined.svg');
+  background-image: url('logo-outlined.svg'), linear-gradient(123deg, rgba(49,53,113,1) 0%, rgba(30,10,67,1) 100%);
```
Comment (lazd) [inline covidatlas/index.css:148]: `background-image: url()` won't work in prod :'(

We'll fix this later, but in the meantime, we have to apply these using `style="background-image: url('`${arc.static('logo-outlined.svg')}');"`

File: src/http/get-index/index.js [hunk 2]
```diff
@@ -38,17 +38,20 @@ ${header('home' /* 'ca-SiteHeader--dark spectrum--dark' */)}
...
       <section class="ca-Section col-xs-12 col-sm-4">
+        <img class="ca-Section-Image" src="${arc.static('icons/COVID_Atlas_Icons_Calendar.svg')}"/>
         <h1 class="spectrum-Heading spectrum-Heading--M">Updated daily</h1>
         <p class="spectrum-Body spectrum-Body--M">Our data updates daily at 9PM PST so you can access the most recent data available.</p>
       </section>
 
       <section class="ca-Section col-xs-12 col-sm-4">
-        <h1 class="spectrum-Heading spectrum-Heading--M">Built by a community</h1>
+      <img class="ca-Section-Image" src="${arc.static('icons/COVID_Atlas_Icons_Community.svg')}"/>
+          <h1 class="spectrum-Heading spectrum-Heading--M">Built by a community</h1>
```
Comment (lazd) [inline src/http/get-index/index.js:54]:
```suggestion
        <h1 class="spectrum-Heading spectrum-Heading--M">Built by a community</h1>
```

**Pull request merged**